pkgs/tools/backup/btrbk/default.nix

```diff
@@ -10,9 +22,9 @@ stdenv.mkDerivation rec {
     sha256 = "153inyvvnl17hq1w3nsa783havznaykdam2yrj775bmi2wg6fvwn";
   };
 
-  nativeBuildInputs = [ asciidoc asciidoctor makeWrapper ];
+  nativeBuildInputs = [ buildPackages.asciidoc buildPackages.asciidoctor makeWrapper ];
```

Comment:
`buildPackages` comes into play whenever the build host is different from the target host. If you were for example cross-compiling a package on an x86 machine with an aarch64 target, then you would need a x86 asciidoc because it runs on the host that is building the packge but an aarch64 perl because the perl binary will be executed on the target host.
In non-cross-platform builds (normal builds with eg `nix-build . -A btrbk` instead of `nix-build . -A pkgsCross.aarch64-multiplatorm.btrbk` ) `buildPackages` is equivalent to `pkgs`.